lib/matplotlib/tests/test_axes.py

```diff
@@ -135,6 +135,48 @@ def test_twinx_cla():
     assert_true(ax.patch.get_visible())
     assert_true(ax.yaxis.get_visible())
 
+@cleanup
+def test_inverted_cla():
+    # Github PR #5450. Setting autoscale should reset 
+    # axes to be non-inverted.
+    # plotting an image, then 1d graph, axis is now down
+    fig = plt.figure(0);
+    ax = fig.gca()
+    # test that a new axis is not inverted per default
+    assert not(ax.xaxis_inverted())
+    assert not(ax.yaxis_inverted())
+    img = np.random.random((100,100))
```

Comment:
thanks for the comment, I have this bad habit of putting semicolons (I squash multiple commands in a line when testing then forget to remove after unsquashing)